README.md

```diff
@@ -69,10 +69,30 @@ AWS command line utility and configure your AWS access credentials.
    that `AWS_DEFAULT_REGION` points to your prefered region.
    These buckets will be created with Terraform, and should not exist before deploying for the first time.
 
+1. Create an AWS ElasticSearch Service instance with the name specified by `$DSS_ES_DOMAIN` (e.g. via `aws es create-elasticsearch-domain --domain-name $DSS_ES_DOMAIN`).
```

Comment:
we should be utilizing terraform for this to maintain a state of the ES instance, see `$DSS_HOME/infra/elasticsearch`

Reply:
#1960